common/app/routes/challenges/utils.js

```diff
@@ -326,12 +326,29 @@ export function filterComingSoonBetaChallenge(
 }
 
 export function filterComingSoonBetaFromEntities(
-  { challenge: challengeMap, ...rest },
+  { challenge: challengeMap, block: blocks, ...rest },
   isDev = false
 ) {
   const filter = filterComingSoonBetaChallenge.bind(null, isDev);
   return {
     ...rest,
+    block: Object.keys(blocks)
+      .map(dashedName => {
+        const block = blocks[dashedName];
+        let challenges = block.challenges;
+        if (Array.isArray(challenges)) {
```

Comment:
Yep, you're right! I originally added this because it seemed to be necessary for something, but it works fine without it now.